perf-tests/clusterloader2/howtorun.md

```diff
@@ -0,0 +1,58 @@
+Process under GCP project. Please remember perf-tests running on kubemark master, so all logs should be checked under /var/log/ on kubemark-master
+
+## Pre-rerequisites: GCP config
+1. run "gcloud version" to ensure your Google Cloud SDK is updated (suggested Google Cloud SDK 298.0.0 and up), Please refer to https://cloud.google.com/sdk/docs/downloads-apt-get or https://cloud.google.com/sdk/docs/downloads-versioned-archives to upgrade your google cloud SDK
```

Comment:
first letter in upper case. applies to all the other paragraphs.

Reply:
updated all paragraphs first letter to upper case